funnel/templates/update_details.html.jinja2

```diff
@@ -7,11 +7,22 @@
 {% block mobile_header_title %}{% endblock %}
 
 {% block left_col %}
+  <div class="flex-wrapper flex-wrapper--center flex-wrapper--space-between nav-btn-wrapper">
+    <a class="mui--text-dark mui--text-body2 nounderline" href="{{ project.url_for('updates') }}">{{ faicon(icon='chevron-double-left-solid', icon_size='body2', css_class='mui--align-middle btn-margin-right') }}{% trans %}All submissions{% endtrans %}</a>
+    <div>
+      {%- if update.getprev() -%}
+        <a href="{{ update.getprev().url_for() }}" class="mui--text-body2 mui--text-capitalize mui--text-dark nounderline prev-btn">{{ faicon(icon=prev_icon, icon_size='subhead', baseline=false, css_class='mui--align-middle btn-margin-right') }}{% trans %}Previous{% endtrans %}</a>
+      {%- endif -%}
```

Comment:
Because the query hits the database, calling `getprev()` twice will hit the database twice. Wrap this in a `with`:

```jinja2
{%- with prev_update = update.getprev() %}{% if prev_update %}
...
```